src/main/java/com/google/cloud/tools/managedcloudsdk/ManagedCloudSdk.java

```diff
@@ -177,6 +179,26 @@ public SdkInstaller newInstaller(Map<String, String> environmentVariables) {
         managedSdkDirectory, version, osInfo, USER_AGENT_STRING, false, environmentVariables);
   }
 
+  /**
+   * Create a new {@link SdkInstaller} with an array corresponding to the --override-components flag
+   * of the installer script.
```

Comment:
Feels weird to mention only `overrideComponents`. How about something along the lines of 

`Create a new {@link SdkInstaller} with gcloud components to install (override) and environment variables to use while running the installer script.`

? (Not sure about the exact wording.)